Change the way Xml content is formatted.

HEskandari opened the pull request:
Resolves #501

@Particular/serviceinsight-maintainers Please review. This changes the way the xml string is formatted and with that addresses the exception in #501. I have moved the formatting method to the ViewModel where I believe it belongs and also added a test (it is not easy to add tests for stuff in the View).

johnsimons commented on src/ServiceInsight/ExtensionMethods/XmlDocumentExtensions.cs:
```diff
@@ -1,24 +1,19 @@
     public static class XmlDocumentExtensions
     {
-        public static string GetFormatted(this XmlDocument document)
+        public static string FormattAsXml(this string xmlContent)
```
@HEskandari too many `t`s, Formatt should be Format ?

HEskandari replied:
Thanks @johnsimons I'll update.

HEskandari commented:
@Particular/serviceinsight-maintainers So I've moved the formatting method into the VM and removed the extension method as agreed. This should be good now.

adamralph commented on src/ServiceInsight/MessageViewers/XmlViewer/XmlMessageViewModel.cs:
```diff
@@ -55,5 +60,24 @@ public void Clear()
+                    return xml.ToString();
+                }
+            }
+            catch (Exception)
```
Aha! This is where the problem is fixed. Previously we were catching `XmlException`.
